### DISCUSSION TITLE
Issue creating new column on Budibase table

### DISCUSSION THREAD

**Discussion by ndlang** [Budibase/budibase#3360] [Help]
I am trying to create a new column on an existing table. When I create the column it redirects me to the create new table page.  I am getting the following response: 
    {"message":"Cannot change the type of a column","status":400}

Budibase Current version: 0.9.186
Firefox version: 94.0.1 (64-bit)

**Comment by ndlang** [comment]
Logs from bbapps:
Trace: BadRequestError: Cannot change the type of a column

    at Object.throw (/app/node_modules/koa/lib/context.js:97:11)

    at /app/dist/api/controllers/table/index.js:80:30

    at Generator.next (<anonymous>)

    at fulfilled (/app/dist/api/controllers/table/index.js:5:58)

    at processTicksAndRejections (internal/process/task_queues.js:97:5)

    at /app/dist/api/index.js:69:21

    at Generator.throw (<anonymous>)

    at rejected (/app/dist/api/index.js:6:65)

    at processTicksAndRejections (internal/process/task_queues.js:97:5)

INFO  [1636818264372] (28 on 7a31b8872c9b): request completed

    res: {

      "statusCode": 200,

      "headers": {

        "vary": "Accept-Encoding",

        "content-type": "application/json; charset=utf-8",

        "content-encoding": "gzip"

      }

    }

**Comment by mjashanks** [comment]
Hi Nathan.

I am unable to reproduce this. Does this happen every time for you? What is your column name and table name? I'm wondering if there is a naming conflict here or a column name that we should not allow.

Can you create new columns of a different name, or on a different table?

If you can't see anything obvious here, feel free to send an export of your app to support@budibase.com - it will include your data, so I understand if this is not possible. If not, then a list of columns in your table & its table name might help.

Thanks!

**Comment by ndlang** [comment]
Answers to your other questions:
Can you create new columns of a different name, or on a different table?
  - I can create columns on other tables, just not on the members table.

**Reply by mjashanks** [reply]
So, could you either
1. Send an export of your app to support@budibase.com (includes all data, so understandable if you don't want to)
or
2. Tell me ..
- the name of your table
- all columns in your table, including types
- the name and type of the column that you are trying to add

Thanks!

**Reply by ndlang** [reply]
Hey i am sorry I didn't respond earlier to this. I actually just rebuilt the internal DB.
